src/signals/incident-management/containers/IncidentDetail/components/SplitNotificationBar/index.js

```diff
@@ -17,43 +52,47 @@ const getErrorMessage = (status) => {
   }
 };
 
+const getDelimiter = (key, length) => {
```

Comment:
Can you add a test for this function?

Reply:
there is a test for two versions already: 
- should render 2  items correctly
- should render 3  items correctly